autoload/db_ui/schemas.vim

```diff
@@ -85,11 +85,27 @@ let s:mysql = {
       \ 'layout_flag': '\\G',
       \ 'quote': 0,
       \ }
+
+let s:oracle_args = 'echo "SET linesize 4000;\nSET pagesize 4000;\nCOLUMN owner FORMAT a20;\nCOLUMN table_name FORMAT a25;\nCOLUMN column_name FORMAT a25;\n%s" | '
```

Comment:
These args hold an SQL query:

```sql
SET linesize 4000;
SET pagesize 4000;

COLUMN owner FORMAT a20;
COLUMN table_name FORMAT a25;
COLUMN column_name FORMAT a25;
```

Without this query, the results of every response from the server are garbled (as seen in a previous comment).